### ISSUE TITLE
Cannot download to /srv, despite having permissions to.

### ISSUE THREAD

**Issue by emwjacobson** [google-deepmind/alphafold3#17]
Hello,

When attempting to download to a subdirectory of `/srv`, I would get an error saying that it needs to be run as root or have permissions for `/srv`, despite having appropriate permissions to write in the subdirectory.

https://github.com/google-deepmind/alphafold3/blob/0341fb7f244779a634702d7cb6e37e540e2db26a/fetch_databases.py#L94-L97

Removing the check resolves the issue and allowed me to download, just thought I would mention it :)

**Comment by Augustin-Zidek** [comment]
Yes, you are right, the check is too strong. All that we need to check is this:

```py
if not os.access(args.download_destination, os.W_OK):
  raise ValueError(f'You do not have permissions to write to {args.download_destination}.')
```

We will send a fix, but the workaround -- as you say -- is to simply remove the check.

Thanks for reporting!

**Comment by Augustin-Zidek** [comment]
Fixed in https://github.com/google-deepmind/alphafold3/commit/44e1fd572a6da02c81e2b3aed94dc9b0d564375c.
